### PR TITLE
Typos - ephemerons

### PR DESCRIPTION
**Pull request by Fourchaux**
Typos - Files ephemeron.ml & ephemeron.mli

### CODE REVIEW

File: stdlib/ephemeron.mli [hunk 1]
```diff
@@ -102,7 +102,7 @@ module K1 : sig
 
   val create: unit -> ('k,'d) t
   (** [Ephemeron.K1.create ()] creates an ephemeron with one key. The
-      data and key are empty *)
+      data and the key are empty *)
```
Comment (damiendoligez) [inline stdlib/ephemeron.mli:105]: Mostly a question of taste here, the original is grammatically correct.

File: stdlib/ephemeron.mli [hunk 2]
```diff
@@ -293,7 +293,7 @@ module Kn : sig
 end
 
 module GenHashTable: sig
-  (** Define hash table on generic containers which have a notion of
+  (** Define an hash table on generic containers which have a notion of
```
Comment (damiendoligez) [inline stdlib/ephemeron.mli:296]: Should be "**a** hash table".

**Comment by Fourchaux** [comment]
Yep : 'a' hash table  
-> ephemeron.mli - L331 : / an hash table / a hash table /

**Comment by damiendoligez** [comment]
merged (commits 5ec086870f9ce002dab4315e140e3f63f1f6a4bf and 4271a4ccef97cded1f2f8f06258a260fd44bf241)

**Pull request closed without merging**